js/components/DeploymentComponent.jsx

```diff
@@ -81,27 +99,27 @@ var DeploymentComponent = React.createClass({
       // Set `title` on cells that potentially overflow so hovering on the
       // cells will reveal their full contents.
       <tr>
-        <td className="overflow-ellipsis" title={model.get("id")}>
-          {model.get("id")}
+        <td className="overflow-ellipsis" title={model.id}>
+          {model.id}
         </td>
         <td>
           <ul className="list-unstyled">
-            {model.get("currentActions").map(function (action) {
+            {lazy(model.currentActions).map(function (action) {
               return <li key={action.app}>{action.app}</li>;
-            })}
+            }).value()}
           </ul>
         </td>
         <td>
           <ul className="list-unstyled">
-            {model.get("currentActions").map(function (action) {
+            {lazy(model.currentActions).map(function (action) {
               return <li key={action.app}>{action.action}</li>;
-            })}
+            }).value()}
```

Comment:
From your link:
```returns //      	Sequence  //   	The new sequence.```
It returns a sequences not an array.

Here's a good blog entry about that and sequences:
http://philosopherdeveloper.com/posts/introducing-lazy-js.html